ios/DevStack.xcodeproj/project.pbxproj

```diff
@@ -581,6 +583,42 @@
 			shellPath = /bin/sh;
 			shellScript = "\"${BUILD_DIR%/Build/*}/SourcePackages/checkouts/firebase-ios-sdk/Crashlytics/run\"\n";
 		};
+		A327766E2BAD960400CC8CF4 /* Generate error messages */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Generate error messages";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "cd \"${PROJECT_DIR}\"\nexec > prebuild-error-messages.log 2>&1 # For debugging\nsh scripts/generate-error-messages.sh\n";
+		};
+		A36C52FD2B594E550055080A /* Copy Moko resources */ = {
```

Comment:
I see this build phase in the `DevStack` target, is it the right place? Can't we have it in `BuildKMP` target as well?